gtsfm/averaging/rotation/cycle_consistency.py

```diff
@@ -26,10 +27,32 @@
 
 
 CYCLE_ERROR_THRESHOLD = 5.0
+MEDIAN_CYCLE_ERROR_THRESHOLD = 10.0
 
 MAX_INLIER_MEASUREMENT_ERROR_DEG = 5.0
 
 
+class EdgeErrorAggregationCriterion(str, Enum):
```

Comment:
Do we need to compare the number of entries for an edge too? Say an edge has just 1 cycle?

Reply:
Do we need to? If its a bad cycle we throw the edge :) I think your point alludes to confidence intervals for the aggregate estimates. I’m not sure if we need that sophistication yet.

Reply:
Thanks for taking a look @ayushbaid.

Hmm we could do that. Although not sure if that's necessary.